ui/js/models/perf/alerts.js

```diff
@@ -84,7 +84,7 @@ treeherder.factory('PhAlerts', [
                 return !alert.is_regression && alert.visible;}),
             'amount_pct').reverse();
             var regressed = _.sortBy(_.filter(this.alerts, function(alert) {
-                return alert.is_regression && alert.visible;}),
+                return alert.is_regression && alert.visible && alert.status != 3;}),
```

Comment:
`alert.status != 3` -> `!alert.isInvalid()` :)

It's better not to use magic numbers here. We actually define getters to see if an alert has various statuses here:

https://github.com/mozilla/treeherder/blob/master/ui/js/models/perf/alerts.js#L63